docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       FIDES__DEV_MODE: "True"
       FIDES__LOGGING__COLORIZE: "True"
       FIDES__USER__ANALYTICS_OPT_OUT: "True"
+      FIDES__CONSENT__TCF_ENABLED: "False"
```

Comment:
It seems like hardcoding this to false is preventing setting tcf_enabled=true in the fides.toml file for local development from taking effect? Can you look at this and verify you can run fides locally with tcf enabled=true?

Reply:
Good catch. I'll look into that now

Reply:
That was causing the issue you described. I removed it from the docker compose file: https://github.com/ethyca/fides/pull/4220/commits/d99ddb024e4296a52cf660317bdd08a8d95823f9

I tried updating it to be `FIDES__CONSENT__TCF_ENABLED: ${FIDES__CONSENT__TCF_ENABLED-}` but that caused validation issues on startup because it would pass in null instead of a boolean value.